src/main/java/com/atlassian/jira/cloud/jenkins/Config.java

```diff
@@ -1,6 +1,24 @@
 package com.atlassian.jira.cloud.jenkins;
 
+import com.google.common.collect.ImmutableMap;
+import io.github.resilience4j.ratelimiter.RateLimiterConfig;
+import io.github.resilience4j.ratelimiter.RateLimiterRegistry;
+
+import java.time.Duration;
+
 public interface Config {
 
     String ATLASSIAN_API_URL = "https://api.atlassian.com";
+    String ATLASSIAN_RATE_LIMITER_CONFIG = "atl";
+
+    RateLimiterConfig DEFAULT_RATE_LIMITER_CONFIG =
```

Comment:
This isn't final config. I've used [these](https://developer.atlassian.com/cloud/jira/platform/user-impersonation-for-connect-apps/#rate-limiting) limits as an example